en/messages.json

```diff
@@ -415,5 +415,34 @@
   },
   "openBookmarkInContainerTab": {
     "message": "Open Bookmark in Container Tab"
+  },
+  "resetSiteCookiesTooltipInfo": {
+    "message": "Reset cookies for this site",
```

Comment:
What does "reset" cookies mean? That's not the terminology that is normally used in Firefox (e.g. "remove").

What's the difference between "Reset cookies" and "Clear cookies"?